beamer/agent/relayer.py

```diff
@@ -29,27 +29,39 @@ def run_relayer_for_tx(
     l2_relay_to_rpc_url: URL,
     privkey: HexBytes,
     tx_hash: HexBytes,
+    prove_tx: bool = False,
 ) -> None:
     relayer = get_relayer_executable()
 
     if not relayer.exists():
         log.error("No relayer found")
         sys.exit(1)
 
+    command_args = [
+        str(relayer),
+        "relay" if not prove_tx else "prove-op-message",
+        "--l1-rpc-url",
+        l1_rpc,
+        "--wallet-private-key",
+        privkey.hex(),
+        "--l2-transaction-hash",
+        tx_hash.hex(),
+    ]
+
+    if prove_tx:
+        command_args.extend(["--l2-rpc-url", l2_relay_from_rpc_url])
```

Comment:
Is `--l2-rpc-url` a new relayer option, specifically for this case? Or is this a typo?

Reply:
no, it is planned like this, so new prove command has its own set of arguments.